code/modules/antagonists/clock_cult/items/clockwork_weapon.dm

```diff
@@ -0,0 +1,108 @@
+/obj/item/twohanded/clockwork
+	name = "Clockwork Weapon"
+	desc = "Something"
+	icon = 'icons/obj/clockwork_objects.dmi'
+	lefthand_file = 'icons/mob/inhands/antag/clockwork_lefthand.dmi';
+	righthand_file = 'icons/mob/inhands/antag/clockwork_righthand.dmi'
+	force_unwielded = 15
+	force_wielded = 5
+	block_power_wielded = 1
+	block_power_unwielded = 0
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = ITEM_SLOT_BACK
+	throwforce = 20
+	throw_speed = 4
+	armour_penetration = 10
+	materials = list(/datum/material/iron=1150, /datum/material/gold=2750)
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "poked", "jabbed", "torn", "gored")
+	sharpness = IS_SHARP_ACCURATE
+	max_integrity = 200
+	var/clockwork_hint = ""
+	var/obj/effect/proc_holder/spell/targeted/summon_spear/SS
+
+/obj/item/twohanded/clockwork/Initialize()
+	. = ..()
+
+/obj/item/twohanded/clockwork/pickup(mob/user)
+	. = ..()
+	user.mind.RemoveSpell(SS)
+	if(is_servant_of_ratvar(user))
+		SS = new
+		SS.marked_item = src
+		user.mind.AddSpell(SS)
+
+/obj/item/twohanded/clockwork/examine(mob/user)
+	. = ..()
+	if(is_servant_of_ratvar(user) && clockwork_hint)
+		. += clockwork_hint
+
+/obj/item/twohanded/clockwork/attack(mob/living/target, mob/living/user)
+	if(!is_servant_of_ratvar(user))
+		to_chat(user, "<span class='large_brass'>You really thought you could get away with that?</span>")
+		return ..(user, user)
+	. = ..()
+	if(!is_reebe(user.z))
+		return
+	if(!QDELETED(target) && target.stat != DEAD && !is_servant_of_ratvar(target) && !target.anti_magic_check(major=FALSE) && wielded)
+		hit_effect(target, user)
+
+/obj/item/twohanded/clockwork/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	. = ..()
+	if(!is_reebe(z))
+		return
+	if(isliving(hit_atom))
+		var/mob/living/target = hit_atom
+		if(!.)
+			if(!target.anti_magic_check() && !is_servant_of_ratvar(target))
+				hit_effect(target, throwingdatum.thrower, TRUE)
+
+/obj/item/twohanded/clockwork/proc/hit_effect(mob/living/target, mob/living/user, thrown=FALSE)
+	return
+
+/obj/item/twohanded/clockwork/brass_spear
```

Comment:
the force values on all of these are a good deal excessive, especially the spear, and considering how clockwork armor's bullet protection nullifies most ranged weapons. Clock cultists already have so many advantages on their home turf

Reply:
They were made to be overpowered since they are significantly outnumbered but I really do think I overdid it a *tad* bit. Will re-lookover balance tomorrow